### PR TITLE
2d.md

### PR DESCRIPTION
**Pull request by AlexBecthold**

### CODE REVIEW

**Comment by noahc** [comment]
Can you squash all these commits? http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html And push back up?

**Comment by AlexBecthold** [comment]
I tried to squash the commits based on what the link you shared told me, but I couldn't quite get it to work.  I'm trying again, but I just come up with an error:

```
stantonstudent@stantonstudent-VirtualBox:~/learn_gosu/alex-becthold$ git rebase -i 2d.md~6
fatal: Needed a single revision
invalid upstream 2d.md~6
```

I have tried it with and without the "~6" but it just comes up with the same error.  At first, it told me I couldn't rebase the commits because I still had one rebase-merge directory open, and that I had to either continue, abort, or skip.  Continue wouldn't work, so I used the abort command and all I get now is the above error.  Is there something simple I'm doing wrong? Or is there still something wrong with the last time I tried to squash the commits?

**Comment by noahc** [comment]
Try  `git rebase -i HEAD~6` instead.

**Comment by AlexBecthold** [comment]
That got me into the editor where I changed all but the first of the "pick" commands to "squash" commands.  Now the website is telling me that another editor will pop up once that is done and saved.  How do I save it?

**Comment by AlexBecthold** [comment]
I used the new HEAD command and it worked, but no other editor came up.  I got an error saying that it could not squash the commit "delete 2d.md".  I couldn't get anything to work again, so opened up a new terminal and when I checked to see what branch I was in, it said

```
stantonstudent@stantonstudent-VirtualBox:~/learn_gosu$ git branch
* (no branch, rebasing Module-1-ab)
  Module-1-ab
  master
```

I'm assuming that this means I am still somehow in the process of rebasing my original commit.  Can I still finish it?

**Comment by noahc** [comment]
Good thought on checking out which branch you're on. Try something like `git rebase --continue`

**Comment by AlexBecthold** [comment]
I believe I may have finally gotten it to work, but for some reason it deleted my radiolab.md and licensing.md files in the process.  Is there any way for me to reverse this process?

**Comment by AlexBecthold** [comment]
Or will I just have to re-do those two files

**Comment by noahc** [comment]
Did you commit them? If so you might be able to use the reflow to retrieve them: https://git-scm.com/docs/git-reflog

**Comment by AlexBecthold** [comment]
I never committed those two files.  I was waiting to get the rebase and squash finished first.  They were just saved in a folder

**Comment by noahc** [comment]
Did you manually delete them? Could you have stashed them? If so, something like `git stash pop` might bring them back. https://git-scm.com/book/no-nb/v1/Git-Tools-Stashing

**Comment by AlexBecthold** [comment]
I noticed that one of the actions that I squashed said "delete file radiolab.md" and "delete file licensing.md".  I thought it was going to delete the old ones that I had committed once before, not the new ones.  I think I'll just have to rewrite them

**Comment by AngryPanda77** [comment]
Those delete lines are just commit messages, so they could be anything. If they did delete anything, you should be able to roll your ref log back. You can also just rewrite them.

File: alex-becthold/licensing.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Simply put, a copyright is the expressed right to use a certain, logo, phrase, image, etc.  For example, the NFL logo is a copyright of the National Football League.  We are affected by copyrights because it limits what we will be able to use to create our game.  We can't use images from another game that is copyrighted, and our game has to be in some way original.  It can't be a copy of another.  In order to stay out of trouble, we need to come up with original ideas for our game and create our own images to use.
```
Comment (noahc) [inline alex-becthold/licensing.md:1]: > In order to stay out of trouble, we need to come up with original ideas for our game and create our own images to use.

Checkout https://creativecommons.org/about/ for why you don't have to create your own images for use, if you don't want to.

**Pull request closed without merging**